recipes/speex/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.2rc2" %}
```

Comment:
@jakirkham when I merged this in, it broke the master branch. Also the feedstock is not created. What can I do? xref #2615 #2618